### PR TITLE
[stable/owncloud] Simplify ingress configuration for cert-manager

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juan@bitnami.com>

#### What this PR does / why we need it:

This PR allows the user to configure the proper annotations when using cert-manager to handle TLS certificates in an easy way. The user just needs to setup a boolean to true to do so

### CODE REVIEW

File: stable/owncloud/templates/ingress.yaml [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
-  name: {{ template "owncloud.fullname" . }}
+  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | trimSuffix "-" -}}"
   labels:
-    app: {{ template "owncloud.fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    release: "{{ .Release.Name }}"
-    heritage: "{{ .Release.Service }}"
-{{- if .Values.ingress.annotations }}
+    app: {{ template "owncloud.fullname" $ }}
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
```
Comment (tompizmor) [inline stable/owncloud/templates/ingress.yaml]: `chart: {{ template "owncloud.chart" $ }}` ?

File: stable/owncloud/values.yaml [hunk 2]
```diff
@@ -21,19 +21,51 @@ image:
...
+  ## The list of hostnames to be covered with this ingress record.
+  ## Most likely this will be just one host, but in the event more hosts are needed, this is an array
+  hosts:
+  - name: owncloud.local
+
+    ## Set this to true in order to enable TLS on the ingress record
+    ## A side effect of this will be that the backend owncloud service will be connected at port 443
+    tls: false
+
+    ## Set this to true in order to add the corresponding annontations for cert-manager
```
Comment (tompizmor) [inline stable/owncloud/values.yaml]: typo

**Review by tompizmor — CHANGES_REQUESTED** [review]

File: stable/owncloud/templates/ingress.yaml (hunk 1 above)
Comment (juan131) [reply stable/owncloud/templates/ingress.yaml]: done

**Review by tompizmor — APPROVED** [review]
/lgtm

**Pull request merged**